### ISSUE TITLE
Errors when triggering deployment

### ISSUE THREAD

**Issue by donalffons** [plasmicapp/plasmic#63]
Hi there and thank you for your great work on Plasmic.

In the last few days, we experienced randomly occurring errors when trying to "publish" our project. One of our team wasn't able to "publish" at all, lately. When I personally use the "publish" feature, I never experienced any issues. Below are screenshots of some of the error messages we got.

![image](https://user-images.githubusercontent.com/33040347/210842812-58352b76-d3ba-487b-8785-fc92afd9d353.png)

![image](https://user-images.githubusercontent.com/33040347/210868809-20bdb8db-6325-4bfd-be0a-62f4c9dda71c.png)

Any help would be much appreciated. Thank you!

**Comment by yang** [comment]
Is there any chance that the permissions in this project have been changed?

Do you experience any issues creating a new Plasmic project and pushing to github?

**Comment by donalffons** [comment]
> Is there any chance that the permissions in this repo have been changed?

No, not that I know of. The "Plasmic App" Github App seems to have all the required permissions. The repository is pretty basic, without major special configurations...
![image](https://user-images.githubusercontent.com/33040347/211005214-7ed0903c-1a01-4679-b3e8-dcd4c96165bc.png)

> Do you experience any issues creating a new Plasmic project and pushing to github?

No. I don't think the problem is with Github. In both screenshots, the error happened in the "Save a new version" stage - not in the "Push to Github" one.

Thanks for looking into this!
